torchrl/objectives/sac.py

```diff
@@ -251,7 +264,7 @@ class _AcceptedKeys:
             state_action_value (NestedKey): The input tensordict key where the
                 state action value is expected.  Defaults to ``"state_action_value"``.
             log_prob (NestedKey): The input tensordict key where the log probability is expected.
-                Defaults to ``"_log_prob"``.
+                Defaults to ``"sample_log_prob"``.
```

Comment:
I renamed this key to make it work with the new API. It did not seem to break anything, but I imagine there was a reason for the naming.